docs/source/index.rst

```diff
@@ -0,0 +1,47 @@
+socksio: Client-side sans-I/O SOCKS proxy implementation
+========================================================
+
+``socksio`` is a sans-I/O library similar to
+`h11 <https://github.com/python-hyper/h11>`_ or
+`h2 <https://github.com/python-hyper/hyper-h2/>`_, this means the
+library itself does not handle the actual sending of the bytes through
+the network, it only deals with the implementation details of the SOCKS
+protocols so you can use it in any I/O library you want.
+
+Current status: alpha
```

Comment:
We can upgrade our status to stable I think, we're pretty confident with this API (as log as you agree!)
If you do agree maybe it's time for v1 :tada:

Can always release a breaking change with semver.

Reply:
Agreed! 🚀